src/index.ts

```diff
@@ -57,7 +56,7 @@ export class CdktfProviderProject extends cdk.JsiiProject {
   constructor(options: CdktfProviderProjectOptions) {
     const {
       terraformProvider,
-      workflowContainerImage = "hashicorp/jsii-terraform",
+      workflowContainerImage,
```

Comment:
Don't we need stuff like python, c# etc?

Reply:
Nope, we currently don't even run on that image 😄 (Which the Projen update added for builds for Python, which actually caused the build to fail).

The `Publisher` Projen component for JSII uses the Github setup-xy actions to setup Python and other deps.

Reply:
Ah that makes sense 👍